README.md

```diff
@@ -332,12 +365,11 @@ $ export GOOGLE_PROJECT=<project_id>
 ```
 
 #### Terraform
-Install manually or [use terraform for latest instructions](https://developer.hashicorp.com/terraform/downloads)
 
 ```console
 $ sudo apt install unzip -y
-$ wget https://releases.hashicorp.com/terraform/1.3.6/terraform_1.3.6_linux_amd64.zip
-$ unzip terraform_1.3.6_linux_amd64.zip
+$ wget https://releases.hashicorp.com/terraform/1.3.6/terraform_1.2.6_linux_amd64.zip
+$ unzip terraform_1.2.6_linux_amd64.zip
```

Comment:
This file is undoing https://github.com/EnterpriseDB/edb-terraform/pull/16 merged changes.
terraform `>=1.3.6` will be required now and README includes link to hashicorp download page.